include/taskolib/Step.h

```diff
@@ -108,16 +111,20 @@ class Step
      *            inappropriate return value for the step type (see above), if a timeout
      *            is encountered, or if termination has been requested via the
      *            communication channel or explicitly by the script.
+     *
+     * \see For more information about step setup scripts see at Sequence.
      */
-    bool execute(Context& context, CommChannel* comm_channel, StepIndex index);
+    bool execute(Context& context, CommChannel* comm_channel, StepIndex index,
+        std::string step_setup = "");
```

Comment:
Hmmm, maybe you are right. Since we are in between the development and productive release we still have the freedom to change it. I will confirm to the wishes to have at least four parameter lists of polymorphic functions.

Reply:
Stuffing the script into the `Context` is not nicer?

Reply:
Yes, I remember this discussion. I would also think it belongs more to the `Context ` instead of `Sequence`. Therefore I moved it to `Context::step_setup`. Simplifies the code a lot.